src/components/shared/input/index.tsx

```diff
@@ -24,6 +24,14 @@ interface Props<TFormValues extends FieldValues> {
   type?: 'text' | 'password';
   autocapitalize?: 'none' | 'sentences' | 'words' | 'characters';
   rightElement?: JSX.Element | JSX.Element[];
+  w?: {
+    base: string;
+    md: string;
+  };
```

Comment:
instead of allowing to pass just the width of the button, let's add a whole custom style prop as,
`style?: StyleProp<TextStyle>`

Reply:
Done.